08-mapping.Rmd

```diff
@@ -249,13 +249,13 @@ source("https://github.com/Robinlovelace/geocompr/raw/master/code/08-tmpal.R", p
 
 Another way to change color settings is by altering color break (or bin) settings.
 In addition to manually setting `breaks` **tmap** allows users to specify algorithms to automatically create breaks with the `style` argument.
-Six of the most useful break styles are illustrated in Figure \@ref(fig:break-styles) and described in the bullet points below:
+Here are six of the most useful break styles:
 
-- `style = pretty`, the default setting, rounds breaks into whole numbers where possible and spaces them evenly.
-- `style = equal` divides input values into bins of equal range, and is appropriate for variables with a uniform distribution (not recommended for variables with a skewed distribution as the resulting map may end-up having little color diversity).
-- `style = quantile` ensures the same number of observations fall into each category (with the potential down side that bin ranges can vary widely).
-- `style = jenks` identifies groups of similar values in the data and maximizes the differences between categories.
-- `style = cont` (and `order`) present a large number of colors over continuous color field, and are particularly suited for continuous rasters (`order` can help visualize skewed distributions).
+- `style = pretty`, the default setting, rounds breaks into whole numbers where possible and spaces them evenly,
```

Comment:
I like semi-colons.

Happy to change them to semi-colons or to leave them as they were as you suggest in your other comments, so that all list can be updated in one go.